index.bs

```diff
@@ -0,0 +1,331 @@
+  :root {
+    --div-info-fg-color: springgreen;
+  }
+}
+div.enum-description > table {
+	border-collapse: collapse;
+	border-top: 2px solid #707070;
+	border-bottom: 2px solid #707070;
+	width: 100%;
+	margin: 2em 0;
+}
+div.enum-description > table > tbody > tr > th,
+div.enum-description > table > tbody > tr > td {
+	padding: 0.2em 0.6em;
+	min-width: 150px;
+	border-top: 1px solid #ddd
+}
+div.enum-description > table > thead > tr > th {
+	line-height: 2em;
+	font-weight: 600;
+	color: var(--div-info-fg-color);
+	border-bottom: 1px solid #707070;
+}
+</style>
+
+<h2 id="introduction">Introduction</h2>
+  Most user-agents have their own mechanisms to block autoplay media, and those
+  mechanisms are implementation-specific. Web developers need to have a way to
+  detect if the autoplay media is allowed or not in order to make actions, such
+  like selecting alternate content or improving the user experience while media
+  is not allowed to autoplay. For instance, if a user-agent only blocks audible
+  autoplay, then web developers can replace audible media with inaudible media
+  to keep media playing, instead of showing a blocked media which looks like a
+  still image to users. If the user-agent does not allow any autoplay media,
+  then web developers could stop loading media resource and related tasks to
+  save the bandwidth and CPU usage for users.
+
+  Currently, this specification only handles {{HTMLMediaElement}} (<{video}>
+  and <{audio}>) and does not handle [[webaudio inline]], [[speech-api inline]]
+  and <{image}> (GIF animation). Although the result from the document's
+  {{Document/autoplayPolicy}} can be used to detect whether web audio can be
+  started if user-agents block web audio to autoplay.
+
+<h2 id="autoplay-detection-api">The Autoplay Detection API</h2>
+  Autoplay detection can be performed on either the {{Document}} element or on
+  the {{HTMLMediaElement}}. The former provides a general idea to authors to
+  know if autoplay media is allowed on this document element. If the page
+  contains multiple documents, an implementation can decide to return different
+  result for each of them. The latter provides more accurate result for whether
+  the autoplay is allowed on a specific media element.
+
+  <h3 id="autoplay-policy">Autoplay Policy Enum</h3>
+    <pre class="idl">
+    enum AutoplayPolicy {
+      "allowed",
+      "allowed-muted",
+      "disallowed"
+    };
+    </pre>
+    <div class="enum-description">
+    <table class="simple" dfn-for="AutoplayPolicy" dfn-type="enum-value">
+      <thead>
+        <tr>
+          <th scope="col" colspan="2">
+            Enumeration description
+      <tbody>
+        <tr>
+          <td>
+            "<dfn>allowed</dfn>"
+          <td>
+            Media are allowed to autoplay.
+        <tr>
+          <td>
+            "<dfn>allowed-muted</dfn>"
+          <td>
+            Inaudible media element</dfn> are allowed to autoplay. An
```

Comment:
Do we need to mention the conditions for web audio to run too?

Reply:
Like "Media and web audio are allowed to autoplay"? Also see this [comment](https://github.com/w3c/autoplay/pull/14/files#r691507834) for a potential problem that media element and web audio has a different result.